bundles/org.openhab.binding.amazonechocontrol/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -1,471 +1,531 @@
-		<state readOnly="false" pattern="%d %unit%"/>
-	</channel-type>
-	<channel-type id="mediaLength" advanced="true">
-		<item-type>Number:Time</item-type>
-		<label>Media Length</label>
-		<description>Media length</description>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-	<channel-type id="startCommand">
-		<item-type>String</item-type>
-		<label>Start</label>
-		<description>Start information (Write only)</description>
-		<state pattern="String" readOnly="false">
-			<options>
-				<option value="Weather">Weather</option>
-				<option value="Traffic">Traffic</option>
-				<option value="GoodMorning">Good morning</option>
-				<option value="SingASong">Song</option>
-				<option value="TellStory">Story</option>
-				<option value="FlashBriefing">Flash briefing</option>
-			</options>
-		</state>
-	</channel-type>
-	<channel-type id="notificationVolume" advanced="true">
-		<item-type>Dimmer</item-type>
-		<label>Notification Volume</label>
-		<description>Notification Volume</description>
-		<state readOnly="false"/>
-	</channel-type>
-	<channel-type id="ascendingAlarm" advanced="true">
-		<item-type>Switch</item-type>
-		<label>Ascending Alarm</label>
-		<description>Ascending alarm up to the configured volume</description>
-		<state readOnly="false"/>
-	</channel-type>
-	<channel-type id="nextReminder" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Next Reminder</label>
-		<description>Next Reminder</description>
-		<state readOnly="true"/>
-	</channel-type>
-	<channel-type id="nextAlarm" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Next Alarm</label>
-		<description>Next alarm</description>
-		<state readOnly="true"/>
-	</channel-type>
-	<channel-type id="nextMusicAlarm" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Next Music Alarm</label>
-		<description>Next music alarm</description>
-		<state readOnly="true"/>
-	</channel-type>
-	<channel-type id="nextTimer" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Next Timer</label>
-		<description>Next timer</description>
-		<state readOnly="true"/>
-	</channel-type>
-</thing:thing-descriptions>
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bindingId="amazonechocontrol"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="account">
+		<label>Amazon Account</label>
+		<description>Amazon Account where the amazon echo devices are registered.</description>
+		<channels>
+			<channel id="sendMessage" typeId="sendMessage"/>
+		</channels>
+		<config-description>
+			<parameter name="discoverSmartHome" type="integer">
+				<default>0</default>
+				<description>Defines if openHAB should discover devices connected to the Amazon Echo.</description>

```

Comment:
shouldn't these parameters have labels?

Reply:
Yes.  And I guess it would be better to make this an enum (fits better to the method you commented on yesterday). I'll check that.